### ISSUE TITLE
[demo_man] highscore traceback - 'NoneType' object has no attribute 'player_list'

### ISSUE THREAD

**Issue by arthurlutz** [missionpinball/mpf-examples#10]
```
'NoneType' object has no attribute 'player_list'
Traceback (most recent call last):
  File "/home/arthur/.virtualenvs/mpf/lib/python3.6/site-packages/mpf/commands/game.py", line 220, in __init__
    MachineController(mpf_path, machine_path, vars(self.args)).run()
  File "/home/arthur/.virtualenvs/mpf/lib/python3.6/site-packages/mpf/core/machine.py", line 709, in run
    self._run_loop()
  File "/home/arthur/.virtualenvs/mpf/lib/python3.6/site-packages/mpf/core/machine.py", line 769, in _run_loop
    raise self._exception['exception']
  File "/usr/lib/python3.6/asyncio/events.py", line 145, in _run
    self._callback(*self._args)
  File "/home/arthur/.virtualenvs/mpf/lib/python3.6/site-packages/mpf/core/async_mode.py", line 39, in _done
    future.result()
  File "/home/arthur/.virtualenvs/mpf/lib/python3.6/site-packages/mpf/modes/high_score/code/high_score.py", line 127, in _run
    if not self.machine.game.player_list:
AttributeError: 'NoneType' object has no attribute 'player_list'
'NoneType' object has no attribute 'player_list'
Traceback (most recent call last):
  File "/home/arthur/.virtualenvs/mpf/lib/python3.6/site-packages/mpf/commands/game.py", line 220, in __init__
    MachineController(mpf_path, machine_path, vars(self.args)).run()
  File "/home/arthur/.virtualenvs/mpf/lib/python3.6/site-packages/mpf/core/machine.py", line 709, in run
    self._run_loop()
  File "/home/arthur/.virtualenvs/mpf/lib/python3.6/site-packages/mpf/core/machine.py", line 769, in _run_loop
    raise self._exception['exception']
  File "/usr/lib/python3.6/asyncio/events.py", line 145, in _run
    self._callback(*self._args)
  File "/home/arthur/.virtualenvs/mpf/lib/python3.6/site-packages/mpf/core/async_mode.py", line 39, in _done
    future.result()
  File "/home/arthur/.virtualenvs/mpf/lib/python3.6/site-packages/mpf/modes/high_score/code/high_score.py", line 127, in _run
    if not self.machine.game.player_list:
AttributeError: 'NoneType' object has no attribute 'player_list'
```

**Comment by arthurlutz** [comment]
Related issue : https://github.com/missionpinball/mpf/issues/1329 maybe there needs something to prevent going into highscore (type "h") when game hasn't started.

**Comment by arthurlutz** [comment]
It is indeed fixed by https://github.com/missionpinball/mpf/commit/191b4f209ee8297576b7604d20ff8287497184ce

**Comment by jabdoa2** [comment]
Backported the fix to 0.52.7
